app/ore/project/factory/ProjectFactory.scala

```diff
@@ -50,6 +50,7 @@ trait ProjectFactory {
   val cacheApi: CacheApi
   val actorSystem: ActorSystem
   val pgp: PGPVerifier = new PGPVerifier
+  val versionRegex = "^[0-9a-zA-Z\\.\\,\\[\\]\\(\\)-]+$".r
```

Comment:
Make this `dependencyVersionRegex`, as it could be talking about project versions.